linker/shared/src/main/scala/org/scalajs/linker/analyzer/Infos.scala

```diff
@@ -389,8 +407,12 @@ object Infos {
 
       methodDef.name match {
         case ComputedName(tree, _) =>
-          traverse(tree)
```

Comment:
The deletion of this `traverse(tree)` is suspicious. Was it done by mistake?

Reply:
Indeed. Fixed.